demo/Demo/Lists.elm

```diff
@@ -0,0 +1,307 @@
+                ]
+            , Options.styled Html.a [ Lists.itemSecondaryAction ] [ Options.styled Html.i [] [ Icon.view "star" [] ] ]
+            ]
+        , Lists.li []
+            [ Options.span [ Lists.itemPrimaryContent ]
+                [ Options.styled Html.i [ Lists.itemAvatar ] [ Icon.view "person" [ Icon.size36 ] ]
+                , Options.span [] [ text "Aaron Paul" ]
+                ]
+            , Options.styled Html.a [ Lists.itemSecondaryAction ] [ Options.styled Html.i [] [ Icon.view "star" [] ] ]
+            ]
+        , Lists.li []
+            [ Options.span [ Lists.itemPrimaryContent ]
+                [ Options.styled Html.i [ Lists.itemAvatar ] [ Icon.view "person" [ Icon.size36 ] ]
+                , Options.span [] [ text "Bob Odenkirk" ]
+                ]
+            , Options.styled Html.a [ Lists.itemSecondaryAction ] [ Options.styled Html.i [] [ Icon.view "star" [] ] ]
+            ]
+        ]
+
+
+avatarsAndControls : Model -> Html Msg
+avatarsAndControls model =
+    Lists.ul [ Options.css "width" "300px" ]
+        [ Lists.li []
+            [ Options.span [ Lists.itemPrimaryContent ]
+                [ Options.styled Html.i [ Lists.itemAvatar ] [ Icon.view "person" [ Icon.size36 ] ]
+                , text "Bryan Cranston"
+                ]
+            , Options.span [ Lists.itemSecondaryAction ]
+                [ Toggles.checkbox Mdl
+                    [ 0 ]
+                    model.mdl
+                    [ Toggles.value True
+                    ]
+                    []
+                ]
+            ]
+        , Lists.li []
+            [ Options.span [ Lists.itemPrimaryContent ]
+                [ Options.styled Html.i [ Lists.itemAvatar ] [ Icon.view "person" [ Icon.size36 ] ]
+                , text "Aaron Paul"
+                ]
+            , Options.span [ Lists.itemSecondaryAction ]
+                [ Toggles.radio Mdl
+                    [ 0 ]
+                    model.mdl
+                    [ Toggles.value False
+                    , Options.css "display" "inline"
+                    ]
+                    []
+                ]
+            ]
+        , Lists.li []
+            [ Options.span [ Lists.itemPrimaryContent ]
+                [ Options.styled Html.i [ Lists.itemAvatar ] [ Icon.view "person" [ Icon.size36 ] ]
+                , text "Bob Odenkirk"
+                ]
+            , Options.span [ Lists.itemSecondaryAction ]
+                [ Toggles.switch Mdl
+                    [ 0 ]
+                    model.mdl
+                    [ Toggles.value True
+                    ]
+                    []
+                ]
+            ]
+        ]
+
+
+twoLines : Model -> Html Msg
+twoLines model =
+    Lists.ul [ Options.css "width" "300px" ]
+        [ Lists.li [ Lists.twoLine ]
+            [ Options.span [ Lists.itemPrimaryContent ]
+                [ Options.styled Html.i [ Lists.itemAvatar ] [ Icon.view "person" [ Icon.size36 ] ]
```

Comment:
Doesn't this produce nested `i` tags (one we made here, one from `Icon.view`)? Couldn't we just do

   [ Icon.view "person" [ Icon.size36, Lists.itemAvatar ] 

?